community/event-checklist.md

```diff
@@ -0,0 +1,96 @@
+toc=true
+
++++
+
+# 1. Logistics - Before the Workshop
+
+✔️Decide the date and time of the event as per the convenience of speakers and audience. Conducting events during exams or some other important event might not be a good idea in general. Checking academic calendar and other events in KGP is suggested.
+
+✔️Publicize the event at least 2 or 3 days before the workshop. Publicizing earlier will fade away the event info from memory and publicizing late will be difficult for audience to plan.
+
+✔️Share the Post via Facebook, Twitter and Linkedin via KOSS Account.
+
+✔️Speakers should be decided atleast a week before the workshop, giving them enough time to prepare.
+
+✔️It is important to have backup speakers for atleast 2 levels.(A backup speaker for the decided speaker. Another backup speaker for the backup speaker)
```

Comment:
A backup for a backup seems like a stretch. Did anything such happen?

Reply:
Recently in a workshop, there was no backup speaker at all. And one of the members had to deliver an impromptu. It was done pretty well actually!

Having at least 2 backup speakers has another important side effect that there will be more people who are aware of the content of the workshop and hence can clarify doubts or can take up in emergency.

In some cases, like the python workshop/git workshop where it happens in 2 rooms, it is good if we have more people.